src/psyclone/psyir/nodes/reference.py

```diff
@@ -191,6 +193,57 @@ def datatype(self):
             return UnresolvedType()
         return self.symbol.datatype
 
+    def previous_access(self):
+        '''
+        :returns: the previous reference to the symbol accessed by this
+                  Reference.
```

Comment:
maybe simpler: `:returns: the previous reference to the same symbol.`

Reply:
Done